libsandbox/libsandbox.c

```diff
@@ -349,14 +349,14 @@ char *egetcwd(char *buf, size_t size)
 
 	/* If tracing a child, our cwd may not be the same as the child's */
 	if (trace_pid) {
-		char proc[20];
-		sprintf(proc, "/proc/%i/cwd", trace_pid);
-		ssize_t ret = readlink(proc, buf, size);
-		if (ret == -1) {
+		char proc[21];
```

Comment:
I'm going to change this to 22 to allow for the minus sign in case trace_pid is somehow negative.

Reply:
Fair.